lib/editor/editor.dart

```diff
@@ -285,6 +288,7 @@ class EditorState extends State<Editor> {
             file = snapshot.data as FileIDE;
 
             if (file.id != currentFileId) {
+              isLoading = true;
               handleFileInit(file);
```

Comment:
As handleFileInit is not awaited here the whole widget is rendered twice, once before the function is run thereby with the old state values and second time after the function is run with the updated state values.